src/HasMedia/HasMediaTrait.php

```diff
@@ -122,7 +122,7 @@ public function hasMedia(string $collectionName = '') : bool
      *
      * @return \Illuminate\Support\Collection
      */
-    public function getMedia(string $collectionName = '', $filters = []) : Collection
+    public function getMedia($collectionName = '', $filters = []) : Collection
```

Comment:
Why not use `string $collectionName = null, $filers = []` and then do `return app(MediaRepository::class)->getCollection($this, $collectionName ?: '', $filters);` for the content here? (Rinse and repeat for the other methods.)